app/models/manifest.rb

```diff
@@ -43,4 +43,15 @@ def self.authorized_search(params, current_user=nil)
     dsl = QueryDSL.new(params: params, current_user: current_user)
     search(dsl)
   end
+
+  def as_public_json(options={})
```

Comment:
I think that with our use of `ManifestSerializer` we shouldn't need this - we can just defined `id` on the serializers as `object.uuid`